packages/playground/src/index.css

```
@@ -16,186 +15,54 @@
-@layer base {
-  * {
-    border-color: var(--border1);
-  }
-  body {
-    background-color: var(--surface1);
-    color: var(--neutral6);
-  }
   html {
     scrollbar-color: var(--color-thumb) var(--color-track);
     scrollbar-width: thin;
   }
```

Stylelint (17.10.0)

[error] Expected empty line before rule (rule-empty-line-before)

(rule-empty-line-before)